test/hybrid.jl

```diff
@@ -17,31 +17,68 @@ import ClimaCore.Domains.Geometry: Cartesian2DPoint
 function hvspace_2D()
     FT = Float64
     vertdomain =
-        Domains.IntervalDomain(FT(-π), FT(π); x3boundary = (:bottom, :top))
+        Domains.IntervalDomain(FT(0), FT(4π); x3boundary = (:bottom, :top))
     vertmesh = Meshes.IntervalMesh(
         vertdomain,
-        Meshes.ExponentialStretching(π / 2),
-        nelems = 20,
+        #Meshes.ExponentialStretching(π / 2),
+        nelems = 64,
     )
     vert_center_space = Spaces.CenterFiniteDifferenceSpace(vertmesh)
+    vert_face_space = Spaces.FaceFiniteDifferenceSpace(vert_center_space)
+
     horzdomain = Domains.RectangleDomain(
         -π..π,
         -0..0,
         x1periodic = true,
         x2boundary = (:a, :b),
     )
-    horzmesh = Meshes.EquispacedRectangleMesh(horzdomain, 20, 1)
+    horzmesh = Meshes.EquispacedRectangleMesh(horzdomain, 10, 1)
     horztopology = Topologies.GridTopology(horzmesh)
 
-    quad = Spaces.Quadratures.GLL{4}()
+    quad = Spaces.Quadratures.GLL{8}()
     horzspace = Spaces.SpectralElementSpace1D(horztopology, quad)
 
-    hvspace = Spaces.ExtrudedFiniteDifferenceSpace(horzspace, vert_center_space)
-    return hvspace
+    hv_center_space =
+        Spaces.ExtrudedFiniteDifferenceSpace(horzspace, vert_center_space)
+    hv_face_space = Spaces.FaceExtrudedFiniteDifferenceSpace(hv_center_space)
+    return (hv_center_space, hv_face_space)
 end
 
-@testset "1D SE, 1D FV Extruded Domain ∇ ODE Solve horzizontal" begin
+@testset "1D SE, 1D FV Extruded Domain ∇ ODE Solve vertical" begin
+
+    hv_center_space, hv_face_space = hvspace_2D()
+    V =
+        Geometry.Cartesian13Vector.(
+            zeros(Float64, hv_face_space),
+            ones(Float64, hv_face_space),
+        )
 
+    function rhs!(dudt, u, _, t)
+        A = Operators.AdvectionC2C(
```

Comment:
Here it seems that that `AdvectionC2C` operates on `Cartesian13Vector`, should we expect the same behavior from the spectral operators, @simonbyrne ?

Reply:
Ok, I can see how we can leverage the implicit Ekman example for the vertical velocity equation here.